### DISCUSSION TITLE
Problems logging in

### DISCUSSION THREAD

**Discussion by bbat2575** [cyllab/CalcUS#25] [Developer support]
Hey devs,

I have everything up and running and am excited to contibute to this repo. However, when I attempt to log in using the email I provided when I generated the .env file, and the password 'default', I get a page that says: "Content could not be loaded: internal server error".

I don't know if this helps, but when I launch the server from the terminal, it mentions these lines:
calcus-postgres-1     | 2023-02-06 06:22:16.719 UTC [37] FATAL:  password authentication failed for user "calcus"
calcus-postgres-1     | 2023-02-06 06:22:16.719 UTC [37] DETAIL:  Password does not match for user "calcus".

And when I try to log in, this comes up in the terminal:
calcus-web-1          | psycopg2.OperationalError: FATAL:  password authentication failed for user "calcus"

Any help would be appreciated. Thanks!

**Comment by RaphaelRobidas** [comment]
Thanks for reporting this behavior. Based on the logs, it seems that the password in the .env file does not match the password of the database. Normally, if no database exists, it will be created with the password in the .env file. Is it possible that a database with another password was previously generated? It will be in the `data` subdirectory.

In any case, if you did not launch any calculation, you can safely delete the `data` subdirectory and recreate an empty `data` subdirectory in order to force the creation of a new database. It should now use the correct password, and it will recreate your account as well.

Let me know if that solves the issue! I'm happy to have you interested in the project.

**Reply by bbat2575** [reply]
That worked like a charm. Thank you so much!!
